android/app/src/main/AndroidManifest.xml

```diff
@@ -69,5 +76,10 @@
             </intent-filter>
         </activity>
     </application>
-
+    <queries>
```

Comment:
Remove `android:host` pointing to Jitsi above on this file to test if it fixes the crash